Resources/Prototypes/_Moffstation/Entities/Clothing/Head/helmets.yml

```
@@ -1,3 +1,18 @@
+- type: entity
+  id: FoldableHelmetBase
+  abstract: true
+  components:
+  - type: Foldable
+    unfoldVerbText: fold-untuck-hair-verb
+    foldVerbText: fold-tuck-hair-verb
+    canFoldInsideContainer: true
+  - type: FoldableClothing
+    foldedHideLayers:
+    - Hair
+    unfoldedHideLayers: [ ]
+  - type: HideLayerClothing
+    layers: [ ]
```

```suggestion
  - type: HideLayerClothing
```
Should work without that